code/nodes/featureDescription/utils.q

```diff
@@ -1,27 +1,33 @@
+// code/nodes/featureDescription/utils.q - Utilities featureDescription node
+// Copyright (c) 2021 Kx Systems Inc
+//
+// Utility functions specific the the featureDescription node implementation
+
 \d .automl
 
 // @private
 // @kind function
 // @category featureDescriptionUtility
-// @fileoverview Apply data from a table relating to a subset of columns to a
-//    list of aggregating functions in order to retrieve relevant statistics to
-//    describe the dataset
-// @param  feature {tab} Feature data as a table
-// @param  colList {sym[]} Column list on which the functions are to be applied
-// @param  funcList {lambda[]} List of functions to apply to relevant data
-// @return {mat[]} Descriptive statistics and information
+// @desc Apply data from a table relating to a subset of columns to a
+//   list of aggregating functions in order to retrieve relevant statistics to
+//   describe the dataset
+// @param  feature {table} Feature data as a table
```

Comment:
Can remove extra space between `@param` and the parameter name for the below parameters